Applications/ReinforcementLearning/DeepQ/jni/main.cpp

```diff
@@ -496,21 +496,22 @@ int main(int argc, char **argv) {
           targetNet.finalize();
           return -1;
         }
-        float *nqa = NQ->getData();
+        const float *nqa = NQ->getData();
 
         /**
          * @brief     Update Q values & udpate mainNetwork
          */
+        nntrainer::Tensor _q = *Q;
         for (unsigned int i = 0; i < in_Exp.size(); i++) {
           if (in_Exp[i].done) {
-            Q->setValue(i, 0, 0, (int)in_Exp[i].action[0],
+            _q.setValue(i, 0, 0, (int)in_Exp[i].action[0],
                         (float)in_Exp[i].reward);
           } else {
             float next = (nqa[i * NQ->getWidth()] > nqa[i * NQ->getWidth() + 1])
                            ? nqa[i * NQ->getWidth()]
                            : nqa[i * NQ->getWidth() + 1];
             try {
-              Q->setValue(i, 0, 0, (int)in_Exp[i].action[0],
+              _q.setValue(i, 0, 0, (int)in_Exp[i].action[0],
```

Comment:
I thought we have coding convention not to use variable name which started with '_'. Because there might be collision with compiler/OS/standard libraries. Beside, It is also not good for auto completion of variables on Editor.

Reply:
Fixed accordingly